release-please-config.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://raw.githubusercontent.com/googleapis/release-please/main/schemas/config.json",
+  "include-component-in-tag": true,
+  "include-v-in-tag": true,
+  "tag-separator": "@",
+  "separate-pull-requests": true,
+  "bump-minor-pre-major": true,
+  "bump-patch-for-minor-pre-major": true,
+  "extra-files": [
+    "README.md",
+    "mkdocs.yml"
+  ],
```

Comment:
What do these `extra-files` do in this context?

Reply:
this was hangover from releasing docs